webgoat-lessons/jwt/src/main/resources/lessonPlans/en/JWT_decode.adoc

```diff
@@ -0,0 +1,12 @@
+== Decoding a JWT token
+
+Let's try decoding a JWT token, for this you can use the webWolfLink:jwt[noLink] inside WebWolf.
```

Comment:
The downside of the [noLink] is that it will make a link that overwrites the WebGoat if you click on it. Maybe it is possible to create here a target=_blank or something as well

Reply:
done